pdc/apps/compose/migrations/0011_remove_duplicate_pathtypes.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from django.db import migrations, models
+
+def remove_duplicate_pathtype(apps, schema_editor):
+    PathType = apps.get_model('compose', 'PathType')
+    ComposeRelPath = apps.get_model('compose', 'ComposeRelPath')
+    qs = PathType.objects.values('name')
+    uniq = []
+    dup = []
+    for x in qs:
+        if x not in uniq:
+            uniq.append(x)
+        else:
+            dup.append(x)
+    dup = [dict(t) for t in set([tuple(d.items()) for d in dup])]
```

Comment:
Sees a little complicate. Looks only use `name ` is enough?
Maybe some thing like:
```
unqi = set([])
dup = set([])
for x in qs:
     if x['name'] not in uniq:
          uniq.add(x['name'])
    else:
          dup.add(x['name'])
    for name in dup:
         ........................................
```